src/routes/DigitalAssets/routes/Receive/modules/digitalAssetsReceive.js

```diff
@@ -19,50 +15,16 @@ export function closeView() {
   }
 }
 
-export function setSearchQuery(query: string) {
-  return {
-    type: SET_SEARCH_QUERY,
-    payload: {
-      query,
-    },
-  }
-}
-
-export function clean() {
-  return {
-    type: CLEAN,
-  }
-}
-
-export type DigitalAssetsManageAction = ExtractReturn<typeof openView> |
-  ExtractReturn<typeof closeView> |
-  ExtractReturn<typeof setSearchQuery> |
-  ExtractReturn<typeof clean>
+export type DigitalAssetsReceiveAction = ExtractReturn<typeof openView> |
+  ExtractReturn<typeof closeView>
 
-const initialState: DigitalAssetsManageState = {
-  searchQuery: '',
-}
+const initialState: {} = {}
 
 const digitalAssetsReceive = (
```

Comment:
but this reducer do nothing and could be easily removed

Reply:
so this module will include only constants and actions - it's ok

Reply:
also, as I see these actions are not used at all.
maybe this module could be removed entirely?

Reply:
Yeah, this unnecessary module was removed